openbook_notifications/helpers.py

```diff
@@ -135,6 +136,64 @@ def send_post_comment_reaction_push_notification(post_comment_reaction):
     _send_notification_to_user(notification=one_signal_notification, user=post_comment_commenter)
 
 
+def send_post_comment_user_mention_push_notification(post_comment_user_mention):
+    mentioned_user = post_comment_user_mention.user
+
+    post_comment_id = post_comment_user_mention.post_comment_id
+    notification_group = 'post_comment_%s' % post_comment_id
+
+    mentioner = post_comment_user_mention.post_comment.commenter
+
+    target_user_language_code = get_notification_language_code_for_target_user(mentioned_user)
+    with translation.override(target_user_language_code):
+        one_signal_notification = onesignal_sdk.Notification(post_body={
+            "contents": {
+                "en": _(
+                    '%(mentioner_name)s · @%(mentioner_username)s mentioned you in a comment.') % {
+                          'mentioner_name': mentioner.profile.name,
```

Comment:
have these translations from backend been generated and uploaded to crowdin,  🤔 
Just merged docs for this 😃

Reply:
nopes, wanted to merge all branches to master before doing this, otherwise some strings might get deleted if they're not in this branch?